README.md

```diff
@@ -84,11 +167,11 @@ Alternatively you can use Port Forwarding. The https://portforward.com/ site has
 ------------------------------------------
 
 ## Exploring the project
-BossRoom is an eight-player co-op RPG game experience, where players collaborate to take down some minions, and then a boss. Players can select between classes that each have skills with didactically interesting networking characteristics. Control model is click-to-move, with skills triggered by mouse button or hotkey. 
+BossRoom is an eight-player co-op RPG game experience, where players collaborate to take down some minions, and then a boss. Players can select between classes that each have skills with didactically interesting networking characteristics. Control model is click-to-move, with skills triggered by mouse button or hotkey.
 
 One of the eight clients acts as the host/server. That client will use a compositional approach so that its entities have both server and client components.
 
-The game is server-authoritative, with latency-masking animations. Position updates are done through NetworkedVars that sync position, rotation and movement speed. NetworkedVars and Remote Procedure Calls (RPC) endpoints are isolated in a class that is shared between the server and client specialized logic components. All game logic runs in FixedUpdate at 30 Hz, matching our network update rate. 
+The game is server-authoritative, with latency-masking animations. Position updates are done through NetworkedVars that sync position, rotation and movement speed. NetworkedVars and Remote Procedure Calls (RPC) endpoints are isolated in a class that is shared between the server and client specialized logic components. All game logic runs in FixedUpdate at 30 Hz, matching our network update rate.
 
 Code is organized into three separate assemblies: **Client**, **Shared** and **Server** which reference each other when appropriate.
```

Comment:
This should be either removed or reworded to reflect Phil's asmdef refactor.

Reply:
done